sdk/servicebus/service-bus/src/experimental/index.ts

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+declare module "../models.js" {
+  interface PeekMessagesOptions {
+    /**
+     * (Experimental for diagnostic purpose) Specifies whether to omit the body when peeking messages. Default value `false`.
```

Comment:
NIT

```suggestion
     * (Experimental for diagnostic purposes) Specifies whether to omit the body when peeking messages. Default value `false`.
```

Reply:
good catch! I prefer not to reset testing for this though...